lib/line/bot/client.rb

```diff
@@ -280,6 +280,24 @@ def get_user_rich_menu(user_id)
         get(endpoint_path)
       end
 
+      # Set default rich menu (Link a rich menu to all user)
+      #
+      # @param rich_menu_id [String] ID of an uploaded rich menu
+      #
+      # @return [Net::HTTPResponse]
+      def set_default_rich_menu(rich_menu_id)
+        endpoint_path = "/bot/user/all/richmenu/#{rich_menu_id}"
+        post(endpoint_path)
```

Comment:
post method is not defined yet.
```
       undefined method `post' for #<Line::Bot::Client:0x00007fb4f1aedc58>
```

Please add following code in lib/line/bot/client.rb
```
+      # Post data, get content of specified URL.
+      #
+      # @param endpoint_path [String]
+      #
+      # @return [Net::HTTPResponse]
+      def post(endpoint_path, payload=nil)
+        raise Line::Bot::API::InvalidCredentialsError, 'Invalidates credentials' unless credentials?
+
+        request = Request.new do |config|
+          config.httpclient     = httpclient
+          config.endpoint       = endpoint
+          config.endpoint_path  = endpoint_path
+          config.credentials    = credentials
+          config.payload        = payload if payload
+        end
+
+        request.post
+      end
+
```

Reply:
Thank you for your advice.
I have added the code !
Please confirm.